shared/rakefiles/xk_util.rake

```diff
@@ -186,32 +186,38 @@ task :display_universal_image_info => [:configure] do
   '", verbose: false
 end
 
-# This task grants the owner role in the current project to the current user
-task :grant_project_admin => [@gcp_creds_file, :configure_extra_tf_vars] do
+task :configure_project_admin_roles do
+  @project_admin_roles = ["roles/iam.serviceAccountTokenCreator"]
```

Comment:
I needed to add `serviceAccountTokenCreator` to `*_project_admin`. While doing so, I noticed that Alfredo's fix in #440 did not modify `revoke_project_admin`, hence this refactor.

However, in my testing the new set of roles was insufficient to deploy a new environment. I added a few things and made some progress, but ultimately gave up. My WIP is in a new PR, XXX.